wzdx_sandbox.py

```diff
@@ -42,9 +109,23 @@ def cmp_status(self, cur_status, prev_status, prev_prev_status):
         prev_status = {k:v for k,v in prev_status['WorkZoneActivity'][0].items() if k not in ignore_keys}
         return cur_status == prev_status
 
-    def update_from_feed(self):
-        r = requests.get(self.feed['url'])
-        wzdx = r.json()['WZDx']
+    def parse_to_json(self, data):
+        format = self.feed['format']
+        if type(data) == dict:
+            out = data
+        elif format == 'xml':
+            xmldict = xmltodict.parse(data)
+            out = json.loads(json.dumps(xmldict))
```

Comment:
I know xml and json aren't perfectly compatible, particularly in things like arrays. Do we think that a generic conversion will be good enough?

Reply:
The generic conversion seemed good enough for the WZDx spec in XML.